src/containers/NavbarTopProfileContainer.js

```
@@ -105,9 +105,7 @@ class NavbarTopProfileContainer extends Component {
               href="/"
               color="primary"
               size="xs"
-              onClick={() =>
-                removeToken().then(() => this.props.onDisconnectedUser())
-              }
+              onClick={() => this.props.onDisconnectedUser().removeToken()}
```

should throw an error